pkg/terraform/executor.go

```diff
@@ -96,8 +96,23 @@ type Executor struct {
 	logger        *log.Entry
 }
 
-// NewExecutor initializes a new Executor.
-func NewExecutor(conf Config) (*Executor, error) {
+// InitializeExecutor initialized Terraform directory using given backend and platform
+// and returns configured executor.
+func InitializeExecutor(assetDir string, verbose bool) (*Executor, error) {
```

Comment:
This helper seems weird. I think Terraform package should have no knowledge about "asset dir".